local/index.js

```diff
@@ -38,11 +38,9 @@ const createQueryParams = () => {
     }
     return params;
 };
-const canonicalUrl = (newQueryParams = {}) => {
-    const currentQueryParams = createQueryParams();
```

Comment:
Are we using this function `createQueryParams` somewhere else or can we delete it?

Reply:
@santios-volusion yes, it us used to create the queryParams in props in the local env.